wallet/node/Main.hs

```diff
@@ -115,6 +116,8 @@ action (WalletNodeArgs (cArgs@CommonNodeArgs{..}) (wArgs@WalletArgs{..})) =
 main :: IO ()
 main = do
     args <- getWalletNodeOptions
-    CLI.printFlags
-    putText "[Attention] Software is built with wallet part"
-    runProduction $ action args
+    let loggingParams = CLI.loggingParams "node" (wnaCommonNodeArgs args)
+    loggerBracket loggingParams . handleAny (usingLoggerName "node" . logError . show) $ do
```

Comment:
`usingLoggerName "wallet"`?

Reply:
Yes, was going to say the same, good idea.